tests/data/dataset_readers/srl_dataset_reader.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=no-self-use,invalid-name
```

Comment:
This file needs to have `test` in its name.  The coverage looks bad because this test isn't actually getting run, because pytest doesn't think it's a test.